sdk/src/psp22.ts

```diff
@@ -90,7 +93,19 @@ export class PSP22 {
     )
   }
 
-  async mint(account: IKeyringPair, value: bigint, block: boolean = true): Promise<TxResult> {
+  async load(address: string) {
+    this.contract = new ContractPromise(this.api, this.abi, address)
+  }
+
+  async mint(
+    account: IKeyringPair,
+    value: bigint,
+    address: string = this.contract.address.toString(),
+    block: boolean = true
+  ): Promise<TxResult> {
+    if (!isLoaded(address, this.contract.address.toString())) {
```

Comment:
It's not the best coding practice. When adding any new method, you must remember to check the load. This makes it harder to extend the code and easy to create bugs when you forget to load.

So, all in all, it will be better to add `setContractAddress`, and the client will change it manually before call